.docker/source/Dockerfile

```diff
@@ -27,4 +27,5 @@ ENV PYTHONIOENCODING UTF-8
 RUN catkin config --extend /opt/ros/$ROS_DISTRO --install --cmake-args -DCMAKE_BUILD_TYPE=Release
     # Status rate is limited so that just enough info is shown to keep Docker from timing out,
     # but not too much such that the Docker log gets too long (another form of timeout)
-    # catkin build --limit-status-rate 0.001 --no-notify
+
+RUN catkin build --limit-status-rate 0.001 --no-notify
```

Comment:
I think, originally this command was part of the previous RUN command.
If you like to separate them by now, you should adjust the indentation of the comment above.

Reply:
Please consider https://github.com/ros-planning/moveit/pull/1207. Although I didn't like it, Dave had a reason to remove the build step.